src/components/Tabs/index.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from '@/components/Tabs/Tabs.module.scss';
+import { TABS } from '@/interfaces/task.type';
+
+type TabsProps = {
```

Comment:
Shouldn't be using string here, I miscommunicated last comment. 
Use `Tab` as type here.
```ts
{
tabs: Tab[];
onSelect: (tab: Tab) => void;
activeTab: Tab;
}
```